shared/constants/fs.js

```diff
@@ -573,18 +578,21 @@ export const userTlfHistoryRPCToState = (
   return I.List(updates)
 }
 
-export const viewTypeFromMimeType = (mimeType: string): Types.FileViewType => {
-  if (mimeType === 'text/plain') {
-    return 'text'
-  }
-  if (mimeType.startsWith('image/')) {
-    return 'image'
-  }
-  if (mimeType.startsWith('audio/') || mimeType.startsWith('video/')) {
-    return 'av'
-  }
-  if (mimeType === 'application/pdf') {
-    return 'pdf'
+export const viewTypeFromMimeType = (mime: ?Types.Mime): Types.FileViewType => {
+  if (mime != null && mime.displayPreview) {
```

Comment:
We should use `!==` here. Also, I think `if (mime && mime.displayPreview)` would also work here.

Reply:
Flow is not happy with those. Iirc because undefined vs null.

Reply:
Hmm but with `mime: ?Mime,` above, `mimeType` can't be `undefined`., but either `null` or `Types.Mime`. Any chance flow issue was on a different function signature?

Reply:
I think `if (mime && mime.displayPreview)` would be the best here, but it's really a nit ...

Reply:
"!==" does not work due to Flow pointing out "Property displayPreview is missing in null or undefined" (?Mime includes undefined), but using the "if(mime ..." works, will use that.

Reply:
Whoa I had no idea `?Mime` covers `undefined`. Thanks for pointing it out!